### ISSUE TITLE
Snake.FieldOfView() redundant parameter

### ISSUE THREAD

**Issue by Northern-Web** [Northern-Web/Sanke#28]
I would like to propose some minor changes within a commit that you've made some days ago (7b7061bc3642409be90b22f5da6d485b47df7d9a). Please take a look at my comments within your commit that I've linked to.

My suggestion is to remove the second parameter of `Snake.FieldOfView(map, blocks)`. 

The `Map.GetBlockSize()` can simply be accessed by the first parameter when used within `Snake.FieldOfView(map, blocks)`.

In addition, it seems to me that the for loop is the only element within `Snake.FieldOfView(map, blocks)` that makes use of the second parameter, `blocks`. 

**In simple terms:**

- Remove the requirement of the second parameter in `Snake.FieldOfView(map, blocks)`
- In the for loop of `Snake.FieldOfView(map, blocks)`, simply reference the conditional using `map.GetBlockSize()` instead of `blocks` which is being passed in as an entirely separate parameter.

**Comment by lassag** [comment]
Fixed in #30 Refactoring
